### PR TITLE
JCRVLT-338 : argLine getting overridden by parent pom and vault-core pom

### PR DESCRIPTION
**Pull request by sonagupt**
argLine needs to be appended rather than overridden.
Not able to install jacoco via cli as the argLine is getting overridden.

### CODE REVIEW

**Comment by sonagupt** [comment]
@tripodsan Please review

File: parent/pom.xml [hunk 1]
```diff
@@ -45,6 +45,7 @@
         <slf4j.version>1.7.6</slf4j.version>
         <test.oak>false</test.oak>
         <test.vm.options>-Doak=$test.oak}</test.vm.options>
+        <argLine>${test.vm.options}</argLine>
```
Comment (tripodsan) [inline parent/pom.xml]: I'm not sure this is correct. the argline might be used by other plugins as well. I'd rather keep it with the surefire plugin.

**Comment by tripodsan** [comment]
maybe you can provide an example on how you want to invoke the filevault build from your jacoco cli?

**Comment by sonagupt** [comment]
I want to use command : mvn clean org.jacoco:jacoco-maven-plugin:0.8.3:prepare-agent verify org.jacoco:jacoco-maven-plugin:0.8.3:report
Please refer : https://github.com/Alfresco/alfresco-sdk/issues/298
https://stackoverflow.com/questions/46489455/append-the-value-of-argline-param-in-maven-surefire-plugin
https://github.com/jacoco/jacoco/issues/597

**Comment by tripodsan** [comment]
@sonagupt and what is the `argLine` you want to pass into the surefire pluigin? i.e. if you would do it manually, what would you change?

**Comment by sonagupt** [comment]
@tripodsan The argLine for a maven project is common for all the plug-ins. All plugins send arguments which are used while executing maven install. So here surefire plugin is overriding the argLine hence I am not able to append anything using jacoco plugin. As mentioned in the articles the best practice is that argLine parameter should be defined as a property of the pom, rather than a configuration of surefire.
Please refer : Alfresco/alfresco-sdk#298
https://stackoverflow.com/questions/46489455/append-the-value-of-argline-param-in-maven-surefire-plugin
jacoco/jacoco#597

**Comment by tripodsan** [comment]
@sonagupt I understand. but please, share you configuration or how you intend to invoke the filevault build.

**Comment by sonagupt** [comment]
@tripodsan :  mvn clean org.jacoco:jacoco-maven-plugin:0.8.3:prepare-agent verify org.jacoco:jacoco-maven-plugin:0.8.3:report

**Comment by tripodsan** [comment]
fixed in https://github.com/apache/jackrabbit-filevault/commit/2ad8f61143a88f3bbc4fe693cccf1888adc2f4c5

@sonagupt please close this PR for me, thanks.

**Pull request closed without merging**